src/applications/vaos/tests/new-appointment/components/ReviewPage/index.cc-request.unit.spec.jsx

```diff
@@ -270,13 +248,6 @@ describe('VAOS <ReviewPage> CC request', () => {
       'Something went wrong when we tried to submit your request and you’ll need to start over. We suggest you wait a day',
     );
 
-    await screen.findByText('307-778-7550');
```

Comment:
For anyone else looking at this: I'm going to create a ticket for this. In practice, we'd never have facility information here to display in the error alert when submitting a CC request. However, we may want to display the facility information for the site that's being used.